src/cms/templates/regions/region_form.html

```diff
@@ -154,12 +162,12 @@ <h3 class="heading font-bold text-white">{% trans 'Extended Settings' %}</h3>
 
                 <!-- Options related to page-instance-specific permissions -->
                 <span class="block p-2 mt-2 text-xl font-bold">{% trans 'Page permissions' %}</span>
-                <div class="p-2">
+                <div class="p-2 pb-4 border-b solid border-gray-400">
```

Comment:
```suggestion
                <div class="p-2">
```
And I'd revert this bottom border because it's the last entry again.